libs/asset_type_rds.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+"""
+Asset types RDS class
+
+Copyright 2020-2021 Leboncoin
+Licensed under the Apache License, Version 2.0
+Written by Nicolas BEGUIER (nicolas.beguier@adevinta.com)
+"""
+
+from .asset_type import AssetType
+
+# Debug
+# from pdb import set_trace as st
+
+class RDS(AssetType):
+    """
+    RDS Asset Type
+    """
+    def __init__(self, name: str, engine: str, url: str='', public: bool=False):
+        AssetType.__init__(self, name, public=public)
```

Comment:
May be changed to use super().__init__(name, public=public) as it is intended for this :)